contxt/auth/cli.py

```diff
@@ -240,10 +263,12 @@ class CliAuth(Auth):
        for the target service, authenticating with the above Auth0 access token
     """
 
-    def __init__(self) -> None:
-        super().__init__(client_id=CLI_CLIENT_ID, client_secret="")
-        self.auth_service = AuthService()
+    def __init__(self, env: str) -> None:
```

Comment:
for backwards-compat:
```suggestion
    def __init__(self, env: str = "production") -> None:
```